### PR TITLE
Initial reproducible setup

### PR DESCRIPTION
**Pull request by ofpiyush**
To try this, create a cluster following the instructions in the README.

Should I create a .sh file/cli client which does these things for you?

If you have a free aws account, it should work.

The docker image should be enough to perform basic k8s tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,90 @@
...
+$ aws s3api create-bucket --bucket $(echo $KOPS_STATE_STORE | cut -d'/' -f3) --create-bucket-configuration LocationConstraint=$S3_BUCKET_LOCATION_CONSTRAINT
+# Create the cluster
+$ kops create cluster --kubernetes-version 1.5.1 --zones ${AVAILABILITY_ZONES} --node-count $NODE_COUNT --node-size $NODE_SIZE --master-size $MASTER_SIZE $CLUSTER_DOMAIN
+
+# PS: the following command will bring the cluster up
+$ kops update cluster $CLUSTER_DOMAIN --yes
+
+# Install heapster and dashboard
+
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/monitoring-standalone/v1.2.0.yaml
```
Comment (avinassh) [inline README.md]: Its better to pull this file from tagged version. Latest release seems to be v1.4.3:

    kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/v1.4.3/addons/monitoring-standalone/v1.2.0.yaml

**Comment by avinassh** [comment]
> Should I create a .sh file/cli client which does these things for you?

Yes, please.

**Review by avinassh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ofpiyush) [reply README.md]: 👍

**Comment by ofpiyush** [comment]
@sgaurav @avinassh updated readme and moved some things to scripts, please check.

**Review by avinassh — APPROVED** [review]

**Pull request merged**